zproject/settings.py

```diff
@@ -977,6 +977,12 @@ def get_secret(key):
 SOCIAL_AUTH_GITHUB_SECRET = get_secret('social_auth_github_secret')
 SOCIAL_AUTH_LOGIN_ERROR_URL = '/login/'
 SOCIAL_AUTH_GITHUB_SCOPE = ['email']
+SOCIAL_AUTH_GITHUB_ORG_KEY = SOCIAL_AUTH_GITHUB_KEY
+SOCIAL_AUTH_GITHUB_ORG_SECRET = SOCIAL_AUTH_GITHUB_SECRET
+SOCIAL_AUTH_GITHUB_TEAM_KEY = SOCIAL_AUTH_GITHUB_KEY
+SOCIAL_AUTH_GITHUB_TEAM_SECRET = SOCIAL_AUTH_GITHUB_SECRET
+SOCIAL_AUTH_GITHUB_ORG_NAME = None
+SOCIAL_AUTH_GITHUB_TEAM_ID = None
```

Comment:
These last two settings should be put as defaults in DEFAULT_SETTINGS and be added to zproject/prod_settings_template.py, since they are user-configurable settings (and not constant-for-all-Zulip-installations system settings, which are what belong in zporject/zproject/settings).